indigo/distribution.yaml

```diff
@@ -4580,6 +4580,45 @@ repositories:
       url: https://github.com/GT-RAIL/jaco_gazebo.git
       version: develop
     status: maintained
+  jaguar:
+    doc:
+      type: git
+      url: https://github.com/gstavrinos/jaguar.git
+      version: kinetic-devel
+    release:
+      packages:
+      - jaguar_control
+      - jaguar_description
+      - jaguar_msgs
+      - jaguar_navigation
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
This is the page that I am following
http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation